starcluster/plugins/ipcluster.py

```diff
@@ -163,7 +165,10 @@ def _write_config(self, master, user, profile_dir):
         f.close()
 
     def _start_cluster(self, master, profile_dir):
-        n_engines = max(1, master.num_processors - 1)
+        if not self.master_engines: 
```

Comment:
if `self.master_engines == 0` won't this branch be taken? i.e. using 0 or None to turn of the master will not work.